src/components/collection/Collection.tsx

```diff
@@ -68,47 +64,50 @@ const Collection = ({ id }: Props) => {
     );
   }
 
-  if (isError) {
+  if (currentMember?.type === AccountType.Guest) {
+    return null;
+  }
+  if (collection) {
     return (
-      <Box id={id} p={5}>
-        <Error code={ERROR_UNEXPECTED_ERROR_CODE} />
-      </Box>
+      <>
+        <UnpublishedItemAlert
+          itemId={id}
+          canRead={canRead}
+          canPublish={canPublish}
+          isPublished={
+            isLoadingPublishedEntry ||
+            (!!itemPublishEntry && !isErrorPublishedEntry)
```

Comment:
I know it's not you, but I would remove the double negation ^^

```suggestion
            (Boolean(itemPublishEntry) && !isErrorPublishedEntry)
```